cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -175,6 +175,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodeList() {
        ++trajectory_id) {
     const auto& single_trajectory_nodes = all_trajectory_nodes[trajectory_id];
     visualization_msgs::Marker marker;
+    marker.ns = "Trajectory " + std::to_string(trajectory_id);
     marker.id = marker_id++;
```

Comment:
My understanding is that this only has to be unique in a namespace. Could we simplify now by removing `marker_id`, initializing to `0` here and doing a `++marked.id` below?